lib/ui/views/profile/user_favourites_view.dart

```diff
@@ -45,6 +48,27 @@ class _UserFavouritesViewState extends State<UserFavouritesView>
               ),
             );
           });
+        } else {
+          return Center(
```

Comment:
This widget is used in multiple places, hence code duplication, can you make a component in `lib/ui/components` named maybe `cv_no_items_widget.dart`, and move this logic there. Obviously, we would expect you to make the component dynamic that takes in the `title`, `buttonText`, `onTap` params to build the widget.

Also, it's better to check if `onTap` is `null` then just remove the button.